spec/system/employee/employee_kudo_manage_spec.rb

```diff
@@ -57,4 +57,28 @@
       expect(page).to have_text('You have used all your kudos')
     end
   end
+
+  context 'when user receives a kudo' do
+    before do
+      create(:employee)
```

Comment:
you are creating an employee but not assigning it to any variable here, in lines below you are not using this employee, but you are using the employee that you created in let in line 4